docs/src/domains/steps/tls.md

```diff
@@ -38,7 +38,7 @@ file=none
 highlight=false
 +++
 
-1. Add the certificate with the following command:
+1. Run the following command:
```

Comment:
💅 

For line 55: "2. To redeploy your production environment, run the following command:"